Content.Server/Nutrition/EntitySystems/SmokingSystem.cs

```diff
@@ -136,7 +136,7 @@ public override void Update(float frameTime)
                     continue;
                 }
 
-                _reactiveSystem.ReactionEntity(containerManager.Owner, ReactionMethod.Ingestion, inhaledSolution);
+                _reactiveSystem.DoEntityReaction(containerManager.Owner, inhaledSolution, ReactionMethod.Ingestion);
```

Comment:
this was changed because for some reason it was using an exact copy of this method that didnt toarray the reagent list so it just concurrent modification exceptioned (see below)